### PR TITLE
Parameterize docker files

### PR DESCRIPTION
**Pull request by fredg02**
@chrisguindon @autumnfound I'd like to simplify the docker builds by parameterizing them instead of having to deal with subfolders for different versions.

As a test, I did this for the drupal-node and the java-api-base docker builds.

@chrisguindon I have also added a parameter for the npm version to the drupal-node Dockerfile, since I noticed that there is a newer version available.

Let me know if you have any concerns or questions.

If you are fine with this change, I will adapt the other docker builds accordingly.

### CODE REVIEW

**Comment by autumnfound** [comment]
No complaints here! Makes adding/updating versions even easier for me :) +1 for Java img changes

File: build.sh [hunk 1]
```diff
@@ -29,12 +29,17 @@ build hugo-node h0.110.0-n18.13.0
 build hugo-node h0.76.5-n12.22.1 latest
 
 build drupal-node d9.5.10-n18.17.1 latest
+# build_arg <image_name> <tag> [build-args] [latest]
+build_arg drupal-node d9.5.10-n18.18.2 "--build-arg DRUPAL_VERSION=9.5.10 --build-arg NODE_VERSION=v18.18.2"
```
Comment (chrisguindon) [inline build.sh:33]: I suggest we now make this the latest.

We use the latest tag in our Drupal sites. If we set this to the latest, we will automatically upgrade on the next rebuild.

//cc @ericpoirier @theodoreb

**Review by chrisguindon — APPROVED** [review]

File: build.sh (hunk 1 above)
Comment (chrisguindon) [reply build.sh:33]: @fredg02 This is great! Thanks for making these changes which I believe will make it easier for us to upgrade.

Comment (fredg02) [reply build.sh]: Fixed :white_check_mark:

**Pull request merged**